modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -730,6 +730,44 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
 ----
 ====
 
+[discrete]
+[id="rosa-sts-account-wide-roles-and-policies-example-cli-output-for-policies-attached-to-a-role_{context}"]
+==== Example CLI output for policies attached to a role 
+
+When a policy is attached to a role, the ROSA CLI displays confirmation output. The output depends on the type of policy.
+
+* If a policy is a trust policy, it outputs the content of the policy.
+** For the target role with policy attached, ROSA CLI outputs the role name and the console URL of it. 
++
+.Target role with policy attached example output
+[source,terminal]
+----
+I: Attached trust policy to role 'testrole-Worker-Role(https://console.aws.amazon.com/iam/home?#/roles/testrole-Worker-Role)': ******************
+----
++
+** If the attached policy is a trust policy, ROSA CLI outputs the content of this policy.
++
+.Trust policy example output
+[source,terminal]
+----
+I: Attached trust policy to role 'test-Support-Role': {"Version": "2012-10-17", "Statement": [{"Action": ["sts:AssumeRole"], "Effect": "Allow", "Principal": {"AWS": ["arn:aws:iam::000000000000:role/RH-Technical-Support-00000000"]}}]} 
+----
+* If a policy is a permission policy, it outputs the name and public link of this policy or the ARN depending on whether or not the policy is an AWS managed policy or non-managed.
```

Comment:
```suggestion
* If the policy is a permission policy, the ROSA CLI outputs the name and public link of this policy or the ARN depending on whether or not the policy is an AWS managed or unmanaged policy.
```

Reply:
ARN, AWS, ROSA CLI -> expand first occurence ??

Reply:
rather than non-managed or unmanaged policy, the appropriate term in AWS is
"customer-managed policy"